test/functional/ex_cmds/mksession_spec.lua

```diff
@@ -113,5 +116,7 @@ describe(':mksession', function()
 
     local expected_cwd = cwd_dir..'/'..tab_dir
     matches('^term://'..pesc(expected_cwd)..'//%d+:', funcs.expand('%'))
+    command('bd!')
+    sleep(100)  -- Make sure the process exits.
```

Comment:
nice fix! could use retry() somehow to remove the timing factor (and perhaps make the test faster).

Reply:
I can't find anything that can be asserted to check if process has exited here.

Reply:
I need to make retry() more ergonomic, but meanwhile a hack like `neq(2, eval('1+1'))` works to check if the process is dead.

Reply:
This `sleep()` is for making the terminal process has exited, so that `helpers.rmdir()` in `after_each()` will not fail.

Reply:
oops I read that `bd!` as `qa!` 😅  Would `jobpid(&channel)` help here?